pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -870,7 +870,7 @@ cdef class PyObjectHashTable(HashTable):
         for i in range(n):
             val = values[i]
             hash(val)
-            if not _checknan(val):
```

Comment:
i think this might be the *only* usage of checknan. can you remove it (from util.pxd) and see?

Reply:
seemed to be.  There was an import to remove in hashtable and the definition in util.  Removing them passed tests on my laptop.  Have pushed new version.